serving/docs/adapters_api.md

```diff
@@ -16,6 +16,7 @@ This is an extension of the [Management API](management_api.md) and can be acces
 ### Register an adapter
 
 `POST /models/{modelName}/adapters`
+`POST /adapters` (only for single model use case)
```

Comment:
I don't think we need make SM specific API public

Reply:
I don't like leaving undocumented features, but I moved this into a comment at the bottom of the page where it is more easily ignored